examples/multiple.js

```diff
@@ -1,8 +1,7 @@
 const mineflayer = require('mineflayer')
-const { pathfinder, Movements } = require('mineflayer-pathfinder')
-const { GoalInvert, GoalFollow } = require('mineflayer-pathfinder').goals
+const { goals, pathfinder, Movements } = require('mineflayer-pathfinder')
 
-mineflayer.multiple = (bots, constructor) => {
+const multiple = (bots, constructor) => {
```

Comment:
It was intended like this in prevision of moving this function to mineflayer. Please keep the namespace.